cmd/hypper/install.go

```diff
@@ -120,6 +119,9 @@ func runInstall(args []string, client *action.Install, valueOpts *values.Options
 		client.SetNamespace(chartRequested, settings.Namespace())
 	}
 
+	// Reinit the config so we set the proper namespace of the Storage Driver
+	ReinitConfigForNamespace(client.Config, client.Namespace, logger)
```

Comment:
Why not directly do `client.Config.Init(settings.RESTClientGetter(), client.Namespace, settings.HelmDriver, logger.Debugf)`?
I find it more readable that a static function that contains no logic, and no changes in the scope of vars/functions it access.

Reply:
Shouldn't we have access to client.Config.Init there, too?

Reply:
we should but if we are gonna reuse this in teh upgrade for example I rather have a common function that sets this.

In any case this does not work as expected as it clears the full KubeClient ¬_¬